cmd/packageImport.go

```diff
@@ -68,12 +71,16 @@ func newPackageImportCommand() *packageImportCommand {
 				return errors.Wrap(err, "compile regex exclude")
 			}
 
+			if fromStdin {
+				importPackageFromStdin()
+				return nil
+			}
```

Comment:
Really like your solution and idea here. Just have a small.. I guess style thing which makes the code flow more in one direction if you will. I'll just paste the whole function in order to avoid confusion. Please give me your feedback if you agree on these changes  : )

```go
RunE: func(cmd *cobra.Command, args []string) error {
	// Compile exclude flag value to regex
	regexExclude, err := regexp.Compile(session.config.Import.Exclude)
	if err != nil {
		return errors.Wrap(err, "compile regex exclude")
	}

	// Associate different import types with their lists of config origins
	importTypesAndOrigins := map[string][]string{
		flagConfig:             configs,
		flagDirectoryStructure: directories,
		flagRepoStructure:      remoteRepos,
		flagPackage:            packages,
		flagCollection:         collections,
	}

	// Other imports have to be skipped in case of a stdin import because they might pollute stdin and
	// potentially make the package import fail.
	if fromStdin {
		importTypesAndOrigins = map[string][]string{flagStdin: {""}}
	}

	// Loop over all import types and try to import packages from their associated config origins
	sw := statuswriter.New()
	sw.Run()
	for importType, origins := range importTypesAndOrigins {
		for _, origin := range origins {
			go importPackage(sw.NewSink(), origin, importType, regexExclude)
		}
	}
	sw.Wait()

	return nil
},
```